attribution-data-file-share/src/main/java/gov/cms/ab2d/attributionDataShare/AttributionDataShare.java

```diff
@@ -0,0 +1,94 @@
+package gov.cms.ab2d.attributionDataShare;
+
+import gov.cms.ab2d.lambdalibs.lib.PropertiesUtil;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Properties;
+
+import com.amazonaws.services.lambda.runtime.Context;
+import com.amazonaws.services.lambda.runtime.LambdaLogger;
+
+public class AttributionDataShare {
+    
+    private static final String filePartialName = "ab2d-beneids_";
+    private static final String fileFormat = ".txt";
+
+    private static final String filePath = "/opt/";
+
+    private static final String SELECT_ALL_FROM_COVERAGE = "SELECT beneficiary_id from public.coverage";
+
+    // Returns a string with the fileFullPath to the file we wrote out.
+    // I.E: "ab2d-beneids_2023-08-16T12:08:56.235-0700.txt"
+    public String handleRequest(Context context) throws IOException {
```

Comment:
`public void handleRequest(InputStream inputStream, OutputStream outputStream, Context context) throws IOException { `

Reply:
I see this was done in a commit you pushed. Thank you.

Maybe this is lambda magic that I do not quite understand but why are InputStream and OutputStream needed?
This could just be lambda magic that I don't quite understand but for example: it does not look like we are ever writing out to outputStream?

Reply:
https://docs.aws.amazon.com/lambda/latest/dg/java-handler.html  Handler interfaces

In our case, we don't use InputStream. OutputStream is like.. logging. it's like writing to console. I would use OutputStream for the most important messages